src/components/Review/ReviewComment.tsx

```diff
@@ -40,13 +40,16 @@ const ReviewComment: React.FC<ReviewCommentProps> = ({ reviewDecisionId, isEdita
 
   const initialComment = data?.reviewDecision?.comment || ''
   return (
-    <Form>
-      <TextArea
-        defaultValue={initialComment}
-        onChange={(_, { value }) => setComment(String(value))}
-        onBlur={() => updateComment({ variables: { reviewDecisionId, comment } })}
-      />
-    </Form>
+    <Container>
+      <Label className="strong-label" size="large" content={strings.LABEL_REVIEW_OVERALL_COMMENT} />
+      <Form.Field id="review-commment-content">
+        <TextArea
+          defaultValue={initialComment}
+          onChange={(_, { value }) => setComment(String(value))}
+          onBlur={() => updateComment({ variables: { reviewDecisionId, comment } })}
+        />
```

Comment:
Can you globally over-ride the TextArea styles to match the design (border, etc.)
And make it full-width? (though this is not global)

Reply:
Done in the following PR